src/solvers/dgmulti/types.jl

```diff
@@ -143,39 +117,6 @@ GeometricTermsType(mesh_type::Curved, element_type::AbstractElemShape) = NonAffi
 # other potential constructor types to add later: Bilinear, Isoparametric{polydeg_geo}, Rational/Exact?
 # other potential mesh types to add later: Polynomial{polydeg_geo}?
 
-"""
-  DGMultiMesh(vertex_coordinates, EToV, dg::DGMulti{NDIMS};
-              is_on_boundary=nothing,
-              periodicity=ntuple(_->false, NDIMS)) where {NDIMS, Tv}
-
-- `vertex_coordinates` is a tuple of vectors containing x,y,... components of the vertex coordinates
-- `EToV` is a 2D array containing element-to-vertex connectivities for each element
-- `dg::DGMulti` contains information associated with to the reference element (e.g., quadrature,
-  basis evaluation, differentiation, etc).
-- `is_on_boundary` specifies boundary using a `Dict{Symbol, <:Function}`
-- `periodicity` is a tuple of booleans specifying if the domain is periodic `true`/`false` in the
-  (x,y,z) direction.
-"""
-# TODO: DGMulti v0.5. Standardize order of arguments, pass in `dg` first
-function DGMultiMesh(vertex_coordinates, EToV, dg::DGMulti{NDIMS};
-                     is_on_boundary=nothing,
-                     periodicity=ntuple(_->false, NDIMS), kwargs...) where {NDIMS}
-  if haskey(kwargs, :is_periodic)
-    # TODO: DGMulti, v0.5. Remove deprecated keyword
-    Base.depwarn("keyword argument `is_periodic` is now `periodicity`.", :DGMultiMesh)
-    periodicity=kwargs[:is_periodic]
-  end
-
-  md = MeshData(vertex_coordinates, EToV, dg.basis)
-  if NDIMS == 1
-    md = StartUpDG.make_periodic(md, periodicity...)
-  else
-    md = StartUpDG.make_periodic(md, periodicity)
-  end
-  boundary_faces = StartUpDG.tag_boundary_faces(md, is_on_boundary)
-  return DGMultiMesh(dg, GeometricTermsType(VertexMapped(), dg), md, boundary_faces)
-end
-
```

Comment:
I'm not sure we want to remove this - there are a few constructors for `DGMultiMesh`. The `cells_per_dimension` version creates a uniform mesh; this one allows for general unstructured mesh construction.

Reply:
I see, thanks. I added it back again in 2992733ac